src/row.js

```diff
@@ -62,6 +62,22 @@ export class Row {
     this.enemyFaction = row[47];
   }
 
+  getComposition = brackets => {
+    // We could have added all 5 classes and then filter out falsy values, but it would not be resilient to "ghost player" bugs
+    let arr = [this.enemyPlayerClass1, this.enemyPlayerClass2];
+    if (brackets.includes('5s'))
+      arr.push(
+        this.enemyPlayerClass3,
+        this.enemyPlayerClass4,
+        this.enemyPlayerClass5
+      );
+    else if (brackets.includes('3s')) arr.push(this.enemyPlayerClass3);
+    return arr
+      .filter(it => !!it)
```

Comment:
Are there known cases where it is falsy? (Questionning the need of filtering here)

Reply:
yes because I'm supporting multiple brackets at the same time, so the table can contain mixed 2s/3s/5s matches. This filter filters out e.g. enemyPlayerClass3 from a 2s match.